layouts/partials/helpers/container.html

```diff
@@ -0,0 +1,17 @@
+{{- if .start }}
```

Comment:
Why do we need custom a helper here?

Reply:
I'm gonna patch it in the main repo and remove it from here later on. It needs a bit more work for the theme though. Just wanted it to work as I wanted it to. This is need so the content fragment would support the padding variable and remove the duplicate code for the container used in the content fragment. 

The code change is also done in the content fragment. I'll push the change to Syna ASAP.